docs/src/Actions/Start.md

```diff
@@ -0,0 +1,78 @@
+# Redirect User to Frequency Access
+
+To start the authentication loop, your application will generate a redirect URL for each user.
+
+## Quick Reference
+
+- Staging-Testnet: `https://testnet.frequencyaccess.com/siwa/login`
+- Production-Mainnet: `https://www.frequencyaccess.com/siwa/login`
```

Comment:
Don't think we want to say login as we don't actually know if it's a login at this point. We have been using `/siwa/start`